test/e2e/self_hosted.go

```diff
@@ -45,6 +45,7 @@ type SelfHostedSpecInput struct {
 	ArtifactFolder        string
 	SkipCleanup           bool
 	Flavor                string
```

Comment:
Unrelated to this change - do we need the block comment for Flavor here too?

Reply:
not sure, a lot of them don't. I'm guessing the ones that have the block comment are the ones that actually default to a flavor in the spec. This one and others just defer to ApplyTemplateAndWait to default to "default".

Reply:
No worries - definitely not a blocker or something to do on this PR.